src/core-interfaces.ts

```diff
@@ -1519,6 +1519,7 @@ export interface IChartPropsChartPie {
 	 * @default 0
 	 */
 	firstSliceAng?: number
+	chartPieExplosion?: number
```

Comment:
Does this protect us from bad inputs without having to add manual inline checks? @tjinauyeung

Reply:
No, these definitions are not checked unfortunately. We're not using any of the TS definitions when using the library.